### PR TITLE
Add Debricked to Free for OSS tool list

### PR DESCRIPTION
**Pull request by joannaqvarnstrom**

### CODE REVIEW

File: pages/Free_for_Open_Source_Application_Security_Tools.md [hunk 1]
```diff
@@ -183,9 +183,21 @@ Free tools of this type:
         dependencies in your GitHub projects. Supports: Java, .NET,
         JavaScript, Ruby, and Python. Your GitHub projects are
         automatically signed up for this service.
+  - Debricked: free for open source projects and/or smaller teams.
```
Comment (kingthorin) [inline pages/Free_for_Open_Source_Application_Security_Tools.md:186]:
```suggestion
  - Debricked: free for open source projects or smaller teams.
```

http://andore.ddns.net

File: pages/Free_for_Open_Source_Application_Security_Tools.md [hunk 2]
```diff
@@ -183,9 +183,21 @@ Free tools of this type:
...
 Commercial tools of this type that are free for open source:
 
+ 
+   - [Debricked](https://debricked.com) - over 90% true positive rate in supported languages
+     - Identifies, fixes and prevents known vulnerabilities through automation without the need
+       to give access to your source code. Read more at [https://debricked.com](https://debricked.com)
+     - Allows for vulnerability management and license compliance in the same tool
+     - Features automated fix pull request to automatically fix vulnerabilities (currently only for javascript)
+     - Features one of the most complete [vulnerability databases](https://app.debricked.com/en/vulnerability-database) 
+     - Github version: [https://github.com/apps/debricked/](https://github.com/apps/debricked/)
```
Comment (kingthorin) [inline pages/Free_for_Open_Source_Application_Security_Tools.md:200]: Should maintain the existing alpha sorting.

Comment (kingthorin) [inline pages/Free_for_Open_Source_Application_Security_Tools.md:200]:
```suggestion
     - GitHub version: [https://github.com/apps/debricked/](https://github.com/apps/debricked/) 
```

File: pages/Free_for_Open_Source_Application_Security_Tools.md [hunk 3]
```diff
@@ -218,6 +230,8 @@ Commercial tools of this type that are free for open source:
       - GitHub version:
         <https://github.com/apps/whitesource-bolt-for-github>
 
+              
+
```
Comment (kingthorin) [inline pages/Free_for_Open_Source_Application_Security_Tools.md:234]: These should be reverted.
```suggestion
```

File: pages/Free_for_Open_Source_Application_Security_Tools.md (hunk 2 above)
Comment (joannaqvarnstrom) [reply pages/Free_for_Open_Source_Application_Security_Tools.md:200]: From my perspective it is in alphabetic order, am I missing something?

Comment (kingthorin) [reply pages/Free_for_Open_Source_Application_Security_Tools.md]: C - Contrast would come first.

Comment (joannaqvarnstrom) [reply pages/Free_for_Open_Source_Application_Security_Tools.md]: Oups! Missed that :)

**Review by kingthorin — APPROVED** [review]
Thanks

**Pull request merged**